### PR TITLE
Feature/rubocop

### PR DESCRIPTION
**Pull request by zaparka**
@leoapost this is the first batch of Rubocop refactoring. It's just model folder.

### CODE REVIEW

File: app/models/area_of_law.rb [hunk 1]
```diff
@@ -67,14 +73,17 @@ class << self
 
   include Rails.application.routes.url_helpers
 
-  scope :search, -> (search){ where('LOWER(name) like ?', "%#{search.downcase}%").order('name ASC') }
+  scope :search, lambda { |search|
```
Comment (leoapost) [inline app/models/area_of_law.rb]: how about to use `do..end` instead of `{...}` for multi-line blocks?

File: app/models/local_authority.rb [hunk 2]
```diff
@@ -17,11 +17,13 @@ class LocalAuthority < ActiveRecord::Base
   has_many :courts, through: :remits
 
   scope :by_name, -> { order('LOWER(name)') }
-  scope :search, ->(query){ where('LOWER(name) like ?', "#{query.downcase}%").limit(10) }
+  scope :search, ->(query) { where('LOWER(name) like ?', "#{query.downcase}%").limit(10) }
 
-  scope :unassigned_for_area_of_law, ->(area_of_law) {
-    joins("LEFT OUTER JOIN (jurisdictions INNER JOIN remits ON jurisdictions.remit_id = remits.id) ON local_authorities.id = jurisdictions.local_authority_id AND remits.area_of_law_id = #{area_of_law.respond_to?(:id) ? area_of_law.id : area_of_law}").
-    where('remits.id IS NULL')
+  scope :unassigned_for_area_of_law, lambda { |area_of_law|
```
Comment (leoapost) [inline app/models/local_authority.rb]: how about to use `do..end` instead of `{...}` for multi-line blocks?

File: app/models/town.rb [hunk 3]
```diff
@@ -14,15 +14,17 @@ class Town < ActiveRecord::Base
...
 
   belongs_to :county
-  
-  default_scope  { order('LOWER(towns.name)') } # ignore case when sorting
 
-  scope :with_county_name, -> {
-  	joins('left outer join counties on counties.id = towns.county_id').select('counties.name as county_name, counties.id as county_id')
+  default_scope { order('LOWER(towns.name)') } # ignore case when sorting
+
+  scope :with_county_name, lambda {
```
Comment (leoapost) [inline app/models/town.rb]: how about to use `do..end` instead of `{...}` for multi-line blocks?

File: app/models/area_of_law.rb [hunk 4]
```diff
@@ -51,8 +51,14 @@ class << self
     end
   end
 
-  attr_accessible :name, :old_id, :slug, :type_possession, :type_bankruptcy, :type_money_claims, :type_children,
-    :type_adoption, :type_divorce, :group_id, :external_link, :external_link_desc
+  attr_accessible :name, :old_id, :slug, :type_possession, :type_bankruptcy,
+    :type_money_claims, :type_children, :type_adoption, :type_divorce,
+    :group_id, :external_link, :external_link_desc
+
+  attr_accessible :name, :old_id, :slug, :type_possession, :type_bankruptcy,
```
Comment (zaparka) [inline app/models/area_of_law.rb]: Well spotted. That's from the rebasing.

File: app/models/area_of_law.rb (hunk 1 above)
Comment (zaparka) [inline app/models/area_of_law.rb]: will do.

**Comment by zaparka** [comment]
@leoapost I pushed fixes from above.

**Pull request merged**